decision_service/rlclientlib/vw_model.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "model_mgmt.h"
+
+namespace reinforcement_learning { namespace model_management {
+  class vw_model : public i_model{
+  public:
+    int init(model_data& data, api_status* status = nullptr) override;
+    int choose_rank(int& action, char* features, int actions[], api_status* status = nullptr) override;
```

Comment:
>int& action, [](start = 20, length = 12)

what is this parameter?

Reply:
It was but I changed it after looking at the vw interface.  No longer there.

---
In reply to: [193495501](https://github.com/JohnLangford/vowpal_wabbit/pull/1495#discussion_r193495501) [](ancestors = 193495501)